### PR TITLE
fixed bugs

### PR DESCRIPTION
**Pull request by Vache95**

### CODE REVIEW

**Review by Aramayis331 — COMMENTED** [review]
Clear consoles

File: src/components/headerMain/index.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const HeaderMain: FC<headerMainProps> = ({ setHeaderToggle, headerToggle, theme,
         <span></span>
       </div>
       <div className="header__content">
-        <ul>
+        {/* <ul>
           <li>
```
Comment (Aramayis331) [inline src/components/headerMain/index.tsx:29]: Remove comments

File: src/router/index.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const Router: FC = () => {
     <Routes>
       <Route path="" element={<PrivateRoute />}>
         <Route path="" element={<HeaderLayout />}>
```
Comment (Aramayis331) [inline src/router/index.tsx:22]: Add  index

**Review by Aramayis331 — APPROVED** [review]
Normal cod

File: src/components/headerMain/index.tsx (hunk 1 above)
Comment (Vache95) [reply src/components/headerMain/index.tsx:29]: ok

File: src/router/index.tsx (hunk 2 above)
Comment (Vache95) [reply src/router/index.tsx:22]: why?

**Pull request merged**